t/APISIX.pm

```diff
@@ -217,6 +217,13 @@ if ($version =~ m/\/apisix-nginx-module/) {
 _EOC_
 }
 
+my $a6_ngx_vars = "";
+if ($version =~ m/\/apisix-nginx-module/) {
+    $a6_ngx_vars = <<_EOC_;
```

Comment:
```suggestion
    $a6_ngx_vars .= <<_EOC_;
```
Would this be better ? If there is a new setting that needs to be applied, we don't need to change it.

Reply:
There is no need to append the str to `$a6_ngx_vars` which is empty if apisix-nginx-module is missing.